### ISSUE TITLE
Coin Flip Command

### ISSUE THREAD

**Issue by InvisibleOS** [python-discord/sir-lancebot#612]
# Description
Coin flipping is a very common game that can be really interesting to play, especially on Discord. It would be a good choice to have it on Sir LanceBot, known for its simple but fun and interesting games.

## Reasoning
This is a simple game that can instantly cure boredom and can also be very engaging at times. It would be used by many on the server and it also deserves to be added along with the other games, such as tic-tac-toe and battleship.

## Proposed Implementation
It is a pretty simple code and the outcome should be as shown in the image below. __This image of course is not the final output, I will improvise the commands and modify the results according to your needs and requirements.__

Here is what the command would be:
Coin flip command: `flip` with the aliases - `toss` and `coinflip`

This command can be used without an argument but adding an argument will take the user's argument into consideration, also concluding who won in the end.

This command will simply be made using the `random` module.

![](https://cdn.discordapp.com/attachments/765599553319206996/817777532489302076/unknown.png)

## Would you like to implement this yourself?

- [ ] I'd like to implement this feature myself
- [x] Anyone can implement this feature

**Comment by Srijandev** [comment]
Sounds like a great idea , I appreciate that :D

**Comment by HassanAbouelela** [comment]
Hello,
Thanks for the suggestions, they sound cool. Could you split these out into two issues though? They seem like separate features.

Be sure to check out our contributing guide for setting up the bot, and the style guide we use, before starting work. They can both be found here:

https://pythondiscord.com/pages/contributing/

**Comment by InvisibleOS** [comment]
Alright, so I'll change this one to the coin flip command and make another one for the rock-paper-scissors command.

I'll also be sure to check out the contributing guide.

**Comment by HassanAbouelela** [comment]
👍 Sounds good to me, I've assigned you here.

**Comment by hypergamer80** [comment]
@InvisibleOS may i collaborate with you on this project? It seems as if you've forgotten about this issue or has other things to do, so if you'd like I could work with you on this, making it easier.

My nickname on the Python Discord server is "wait... it's all ᓚᘏᗢ? | 🐈"

Best regards.

**Comment by InvisibleOS** [comment]
Hello,
I have already made both the commands (rock-paper-scissors and coin-flip) long ago. I have also shared the code in #dev-log but I didn't receive a definite reply which was why I accidentally forgot about it. I'm ready with the code now, I just need some help getting it added to Sir Lancebot.
Maybe we can discuss further on PyDis, I'll hit you up.

**Comment by JakeM0001** [comment]
@InvisibleOS Hey so is this ready to be implemented, if so then feel free to make a pull request for the command so we can get it pushed.

**Comment by Xithrius** [comment]
@InvisibleOS Can you make a PR with your feature so we can start reviewing it?

**Comment by TizzySaurus** [comment]
@InvisibleOS has stated on discord that they'd like someone else to take over, so I'll do so. Have reassigned the issue accordingly
